tests/rt_utils.sh

```diff
@@ -270,9 +270,25 @@ check_results() {
         d=$( cmp ${RTPWD}/${CNTL_DIR}/$i ${RUNDIR}/$i | wc -l )
 
         if [[ $d -ne 0 ]] ; then
-          echo ".......NOT OK" >> ${REGRESSIONTEST_LOG}
-          echo ".......NOT OK"
-          test_status='FAIL'
+
+          if [[ ${MACHINE_ID} =~ orion || ${MACHINE_ID} =~ hera ]]; then
+            printf ".......NOT OK...TRY ALT CHECK.." >> ${REGRESSIONTEST_LOG}
+            printf ".......NOT OK...TRY ALT CHECK"
+            d=$( ${PATHRT}/compare_ncfile.py ${RTPWD}/${CNTL_DIR}/$i ${RUNDIR}/$i 2>/dev/null | wc -l )
+            if [[ $d -ne 0 ]]; then
+              echo ".....NOT OK" >> ${REGRESSIONTEST_LOG}
+              echo ".....NOT OK"
+              test_status='FAIL'
+            else
+              echo "....OK" >> ${REGRESSIONTEST_LOG}
+              echo "....OK"
+            fi
+          else
+            echo ".......NOT OK" >> ${REGRESSIONTEST_LOG}
+            echo ".......NOT OK"
+            test_status='FAIL'
+          fi
+
```

Comment:
There is too much logic of OK/NOT OK going on here.
It can be cleaned up to a single OK/NOT OK message.